### PR TITLE
wait longer for build log stream

### PR DESCRIPTION
**Pull request by zhoufenqin**
---
We will schedule build job to legion later, it need more time to running a container in legion, so wait longer time for build container logstream

image

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
az containerapp create/update/up --source/--artifact xxx

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/containerapp/HISTORY.rst [hunk 1]
```diff
@@ -11,6 +11,7 @@ upcoming
 * 'az containerapp env telemetry otlp remove': Support remove environment otlp configuration with --otlp-name
 * 'az containerapp env telemetry otlp show': Support show environment otlp configuration with --otlp-name
 * 'az containerapp env telemetry otlp list': Support show environment otlp configurations
+* Wait long time for logstream of Cloud Build to the 'az containerapp up'/'az containerapp create' and 'az containerapp update' commands
```
Comment (Greedygre) [inline src/containerapp/HISTORY.rst:14]: Please add what is the commands related to this change.

Comment (Greedygre) [reply src/containerapp/HISTORY.rst:14]: For example:
'az containerapp up/create/update': Wait longer time for logstream of Cloud Build to make sure the container start

Comment (zhoufenqin) [reply src/containerapp/HISTORY.rst]: good, fixed

**Comment by Greedygre** [comment]
Please rerun related tests with commands:

`azdev test {your test case/file} --live`

You can add `--no-exitfirst` when running multiple tests.

**Review by yilims — APPROVED** [review]

**Review by Greedygre — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**